install/ui/src/freeipa/translations.js

```diff
@@ -0,0 +1,82 @@
+/*  Authors:
+ *    Petr Vobornik <pvoborni@redhat.com>
+ *
+ * Copyright (C) 2013 Red Hat
+ * see file 'COPYING' for use and warranty information
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+define([
+    './jquery',
+    './ipa',
+    './_base/i18n'
+   ],
+   function( $, IPA, i18n) {
+
+ /**
+  *
+  * Retrieve translations from server
+  *
+  * @class translations
+  * @singleton
+  *
+  */
+
+var translations = {};
+
+var retrieve  = function() {
+
+    var result = false;
+
+    var jsondata = {
+        method: "i18n_messages",
+	params: [ [], { "version": window.ipa_loader.api_version } ]
+    };
+
+    var request = {
+        method: 'POST',
+        url: '/ipa/i18n_messages',
```

Comment:
Please move the URL to config.js like any other URL so they can be customized

Reply:
Thank you.